src/plays/dad-jokes/components/categoryModal/CategoryModal.css

```diff
@@ -0,0 +1,43 @@
+/*----------------------------Category Modal Container----------------------------*/
+.category-modal-container {
```

Comment:
Here use the unique class name as a prefix to all the styles... if your unique class name is `dad-jokes-play` all styles should be defined as

`.dad-jokes-play .category-modal-container`

Do it for all the styles. This will make sure no accidental overrides with anything else.